ProcessMaker/processmaker
# FOUR-9610: Duplicate Project Assets

## Issue & Reproduction Steps
As a Solution Developer, I want the ability to duplicate an asset while retaining all of its characteristics, ensuring that I can make edits without affecting other projects.

## Solution
- Implement asset duplication within the Project

## How to Test
- Checkout to the FOUR-9610 branch in Core and Package Projects
- Create a few Projects 
- Add an Asset to the Project
- Navigate to the Project
- Click on the Duplicate within the Elipsis Menu 
- Fill up the inputs and click on Duplicate
----------------
- Please ensure that the Asset has been duplicated.
- If a Process has Screens and Scripts, those assets should be duplicated and added to the project.
- We can also Duplicate assets to multiple projects.
- Please ensure that the unit test passes as well.
-` phpunit vendor/processmaker/package-projects/tests/Models/AssetsTest.php --filter=testDuplicateAssetFromProject`


## Related Tickets & Packages
- Ticket [FOUR-9610](https://processmaker.atlassian.net/browse/FOUR-9610)
- Package Projects

## CI 
ci:package-projects:task/FOUR-9610
ci:next
ci:deploy



## Code Review Checklist


[FOUR-9610]: https://processmaker.atlassian.net/browse/FOUR-9610?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
